src/ddbToEs/ddbToEsHelper.test.ts

```diff
@@ -19,29 +20,91 @@ describe('DdbToEsHelper', () => {
         esMock.clearAll();
     });
 
-    describe('createIndexIfNotExist', () => {
-        test('Create index and alias for new index', async () => {
-            process.env.ENABLE_ES_HARD_DELETE = 'true';
+    describe('createIndexAndAliasIfNotExist', () => {
```

Comment:
Are there test cases for testing the tenant alias format? (I couldn't find one) i.e. looking for the test cases that follow our tenanted format `generateAlias`

Reply:
I added a couple tests for the `generateAlias` method.

It's worth noting that `createIndexAndAliasIfNotExist` no longer decides the alias naming convention, it just takes aliases and indices names as parameters so the method doesn't have multi-tenancy specific code.